### DISCUSSION TITLE
Question of Update DBeaver

### DISCUSSION THREAD

**Discussion by MisterHuangKai** [dbeaver/dbeaver#10874] [Q&A] [answered]
I've looked all over the Discussions, but no useful information was found. If it has a similar question before, I'm so sorry to ask this question.
I used DBeaver for about two weeks. DBeaver7.3.1
Recently, when I opened DBeaver, it reminds me a new version is available.DBeaver7.3.2
It popped up a dialog and showed me three choices,"Upgrade..","More Information...","Close". But whether I chose "Upgrade..." and "More Information..."，it opened a new web page that it's DBeaver website, to let me download a new version installer(zip etc.).  I needed to uninstall and install it.
How to upgrade directly ? Like some game client with one click upgrade.
Or it's my problem ? something wrong operation?(..•˘_˘•..)

**Comment by uslss** [comment, marked as answer]
Unfortunately, it is not implemented yet.
Will be implemented in the scope of #6558

**Reply by wathika-eng** [reply]
3 years later

![Screenshot From 2024-11-07 10-05-11](https://github.com/user-attachments/assets/6ba37aee-2005-4fa1-97a8-d83856ac1ee4)
